include/dump_offload.hpp

```diff
@@ -408,14 +409,15 @@ inline void requestRoutes(App& app)
                 << " offload initiated by: " << conn.req.session->clientIp;
             bmcHandlers[&conn]->getDumpSize(dumpId, dumpType);
         })
-        .onclose([](crow::streaming_response::Connection& conn) {
+        .onclose([](crow::streaming_response::Connection& conn, bool&) {
             auto handler = bmcHandlers.find(&conn);
             if (handler == bmcHandlers.end())
             {
                 BMCWEB_LOG_DEBUG << "No handler to cleanup";
                 return;
             }
             handler->second->cleanupSocketFiles();
+            handler->second->resetOffloadURI();
```

Comment:
no need to check for the completion status here before resetOffloadURI ?

Reply:
For BMC dumps, pldmd is not involved so bmcweb has to resetOffloadURI for both success/failure

Reply:
Success case can be  added dump manager

Reply:
Dhruv/sunitha

for bmc dumps, I have  change to reset offloadURI only in error cases,
in success case, dump manager reset offloadURI for bmc dump and host stored dumps
error cases, bmcweb resets